app/views/posts/index.html.erb

```diff
@@ -29,7 +28,7 @@
     <% if post.comments.count > 0 %>
       <div class="comments">
         <ul>
-            <% post.comments.order(created_at: :desc).limit(5).each do |comment| %>
+            <% post.recent_comments.each do |comment| %>
                 <li><%= comment.author.name %>: <%= comment.text %></li>
             <% end %>
         </ul>
```

Comment:
- [ ] Kindly style the post index page so that the profile picture and the user details are centered and closer to each other as we have [in the wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAACD5D766UO6XCJRMJJJ5SEVSZEJ22WQ).

  ![image](https://github.com/codedit334/Blog_App/assets/36542986/97d5580b-93fd-44f9-8816-633773d7a22c)